.github/workflows/helm.yaml

```diff
@@ -25,17 +25,12 @@ jobs:
         with:
           version: v3.5.0
```

Comment:
Would you please confirm that we can remove this section because Helm is installed by `make setup`?

Reply:
removed.